.github/workflows/coverage-report.yml

```diff
@@ -0,0 +1,59 @@
+name: Regression Tests Coverage Report
+
+on:
+  pull_request_target:
+    paths:
+    - "examples/**"
+    - "test/**"
+
+jobs:
+  report:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: refs/pull/${{ github.event.pull_request.number }}/head
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run coverage report
+        run: |
+          node test/util/report.js >> coverage.log || true
+
+      - name: Comment on PR
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const fs = require('fs');
+            const commentBody = '```' + fs.readFileSync('coverage.log', 'utf8') + '```';
+
+            if (commentBody.length === 0) {
+              return
+            }
+            // Get the existing comments.
+            const {data: comments} = await github.issues.listComments({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.payload.number,
+            })
+            // Find any comment already made by the bot.
+            const botComment = comments.find(comment => comment.user.id === 41898282)
+
+            if (botComment) {
+              await github.issues.updateComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                comment_id: botComment.id,
+                body: commentBody
+              })
```

Comment:
Does this replace the comment, not append? Because it we don't replace it will get really long! Seems like it from the documentation but just wanted to be sure.

Reply:
It replaces it if it exists. You can see it in the "edit" dropdown on my test PR in my fork https://github.com/nschonni/aria-practices/pull/19#issuecomment-688025952